lib/query_plan_hints.rb

```diff
@@ -0,0 +1,17 @@
+module QueryPlanHints
+
+  # Method decorator to ensure not using a specific query plan
+  # during postgres privilege deriviations
+  def disable_mergejoin(method_name)
+    if ActiveRecord::Base.connection.is_a? ActiveRecord::ConnectionAdapters::PostgreSQLAdapter
```

Comment:
Is there a way to use `PolicyMachineStorageAdapter::ActiveRecord::Assignment` instead of `ActiveRecord::Base` here and below? Not sure the initialization timing works out of the box, but it'd play better with projects that use multiple database connections.